pycbc/io/live.py

```diff
@@ -267,15 +267,18 @@ def __init__(self, coinc_ifos, ifos, coinc_results, **kwargs):
                                                     coinc_inspiral_row.snr,
                                                     min(eff_distances),
                                                     kwargs['mc_area_args'])
-            kwargs['hasmassgap_args'] = copy.deepcopy(kwargs['mc_area_args'])
-            kwargs['hasmassgap_args']['mass_gap'] = True
-            kwargs['hasmassgap_args']['mass_bdary']['ns_max'] = 3.0
-            kwargs['hasmassgap_args']['mass_bdary']['gap_max'] = 5.0
-            self.hasmassgap = calc_probabilities(
-                                  coinc_inspiral_row.mchirp,
-                                  coinc_inspiral_row.snr,
-                                  min(eff_distances),
-                                  kwargs['hasmassgap_args'])['Mass Gap']
+            if 'embright_mg_max' in kwargs['mc_area_args']:
+                kwargs['hasmg_args'] = copy.deepcopy(kwargs['mc_area_args'])
+                kwargs['hasmg_args']['mass_gap'] = True
+                kwargs['hasmg_args']['mass_bdary']['gap_max'] = \
+                    kwargs['mc_area_args']['embright_mg_max']
+                self.hasmassgap = calc_probabilities(
+                                      coinc_inspiral_row.mchirp,
+                                      coinc_inspiral_row.snr,
+                                      min(eff_distances),
+                                      kwargs['hasmg_args'])['Mass Gap']
```

Comment:
Is it necessary to store the args as a new key in the `kwargs` dict? It seems to me you could just use a local variable `hasmg_args`, I think that would make the code more readable.